app/views/YourFinancialInstitutionsView.scala.html

```diff
@@ -16,12 +16,35 @@
 
 @this(
         layout: templates.Layout,
-        govukButton: GovukButton
+        formHelper: FormWithCSRF,
+        govukRadios: GovukRadios,
+        govukButton: GovukButton,
+        govukSummaryList: GovukSummaryList,
 )
 
-@()(implicit request: Request[_], messages: Messages)
+@(form: Form[_], institutions: SummaryList)(implicit request: Request[_], messages: Messages)
 
-@layout(pageTitle = titleNoForm(messages("yourFinancialInstitutions.title"))) {
-   <!--TODO: This is a placeholder page. Will be implemented as part of DAC6-3048-->
-    <h1 class="govuk-heading-xl">@messages("yourFinancialInstitutions.heading")</h1>
+@layout(pageTitle = title(form, messages("yourFinancialInstitutions.title"))) {
+    @if(institutions.rows.isEmpty) {
+        <h1 class="govuk-heading-xl">@messages("yourFinancialInstitutions.heading.noFinancialInstitutions")</h1>
```

Comment:
minor - we can use the Heading component to tidy the html up here

Reply:
sorry @C-Thomp , I have to leave early so I couldn't get a chance to update it. As I am away next week, please feel free to make any updates you like